utils/utils.go

```diff
@@ -8,6 +8,21 @@ import (
 
 // Helper utility functions //
 
+func ErrorMessageContainsAny(err error, msgs ...string) bool {
+	if err == nil {
+		return false
+	}
+
+	errMsg := err.Error()
+	for _, str := range msgs {
+		if strings.Contains(errMsg, str) {
```

Comment:
use https://pkg.go.dev/errors#Is instead of comparing the error strings. This gives better type checking.

Reply:
[97c51a0](https://github.com/Layr-Labs/eigenda-proxy/pull/140/commits/97c51a0994e1a02077ce6f6a432a1ff6e44d8fd4)